archiver/src/main/java/com/netflix/conductor/archiver/config/AppConfig.java

```diff
@@ -41,6 +41,10 @@ public int keepDays() {
 		return config.getInt("keep_days", 30);
 	}
 
+	public String cleanupMessageWorkflows() {
```

Comment:
I am thinking, maybe better to return an array of strings (workflows) and internally read variable - if empty return predefined array otherwise split and return. In that case there will be only one place for read/action.